aireview/justfile

```diff
@@ -15,4 +13,47 @@ run:
     poetry run python src/aireview/main.py
 
 tests:
-    poetry run pytest
\ No newline at end of file
+    poetry run pytest
+
+prefect-login:
```

Comment:
**style**: The docker-push recipe resets DOCKER_BUILDKIT but doesn't use it. Consider removing if not needed for the push operation.